examples/bigquery-analyze-realtime-reddit-data/terraform/main.tf

```diff
@@ -0,0 +1,284 @@
+    "name": "subreddit",
+    "type": "STRING",
+    "mode": "NULLABLE",
+    "description": ""
+  },
+  {
+    "name": "author",
+    "type": "STRING",
+    "mode": "NULLABLE",
+    "description": ""
+  },
+  {
+    "name": "comment_text",
+    "type": "STRING",
+    "mode": "NULLABLE",
+    "description": ""
+  },
+  {
+    "name": "distinguished",
+    "type": "BOOLEAN",
+    "mode": "NULLABLE",
+    "description": ""
+  },
+  {
+    "name": "submitter",
+    "type": "BOOLEAN",
+    "mode": "NULLABLE",
+    "description": ""
+  },
+  {
+    "name": "total_words",
+    "type": "STRING",
+    "mode": "NULLABLE",
+    "description": ""
+  },
+  {
+    "name": "reading_ease_score",
+    "type": "STRING",
+    "mode": "NULLABLE",
+    "description": ""
+  },
+  {
+    "name": "reading_ease",
+    "type": "STRING",
+    "mode": "NULLABLE",
+    "description": ""
+  },
+  {
+    "name": "reading_grade_level",
+    "type": "STRING",
+    "mode": "NULLABLE",
+    "description": ""
+  },
+  {
+    "name": "sentiment_score",
+    "type": "STRING",
+    "mode": "NULLABLE",
+    "description": ""
+  },
+  {
+    "name": "censored",
+    "type": "STRING",
+    "mode": "NULLABLE",
+    "description": ""
+  },
+  {
+    "name": "positive",
+    "type": "STRING",
+    "mode": "NULLABLE",
+    "description": ""
+  },
+  {
+    "name": "neutral",
+    "type": "STRING",
+    "mode": "NULLABLE",
+    "description": ""
+  },
+  {
+    "name": "negative",
+    "type": "STRING",
+    "mode": "NULLABLE",
+    "description": ""
+  },
+  {
+    "name": "subjectivity_score",
+    "type": "STRING",
+    "mode": "NULLABLE",
+    "description": ""
+  },
+  {
+    "name": "subjective",
+    "type": "STRING",
+    "mode": "NULLABLE",
+    "description": ""
+  },
+  {
+    "name": "url",
+    "type": "STRING",
+    "mode": "NULLABLE",
+    "description": ""
+  },
+  {
+    "name": "comment_date",
+    "type": "STRING",
+    "mode": "NULLABLE",
+    "description": ""
+  },
+  {
+    "name": "comment_timestamp",
+    "type": "STRING",
+    "mode": "NULLABLE",
+    "description": ""
+  },
+  {
+    "name": "comment_hour",
+    "type": "STRING",
+    "mode": "NULLABLE",
+    "description": ""
+  },
+  {
+    "name": "comment_year",
+    "type": "STRING",
+    "mode": "NULLABLE",
+    "description": ""
+  },
+  {
+    "name": "comment_month",
+    "type": "STRING",
+    "mode": "NULLABLE",
+    "description": ""
+  },
+  {
+    "name": "comment_day",
+    "type": "STRING",
+    "mode": "NULLABLE",
+    "description": ""
+  }
+]
+EOF
+
+}
+
+resource "google_storage_bucket" "app_bucket" {
+    name          = "${var.app_bucket}"
+    location      = "US"
+    force_destroy = true
+}
+
+resource "google_dataflow_job" "pubsub_stream" {
+    name = "ps-to-bq-${var.pubsub_topic_name}"
+    template_gcs_path = "gs://dataflow-templates-us-central1/latest/PubSub_to_BigQuery"
+    temp_gcs_location = "gs://${google_storage_bucket.app_bucket.name}/dataflow/tmp"
+    region = "us-central1"
```

Comment:
would be cool to have that in the vars, feel free to use us-central1 as a default